.github/CONTRIBUTING.md

```diff
@@ -184,15 +184,15 @@ In the above example, we made our health_loss variable a per second value rather
 For example, if SSmobs is set to run once every 4 seconds, it would call process once every 4 seconds and multiply your health_loss var by 4 before subtracting it. Ensuring that your code is frame independent.
 
 ### No overriding type safety checks
-The use of the : operator to override type safety checks is not allowed. You must cast the variable to the proper type.
+The use of the : operator to override type safety checks is not allowed. You should cast the variable to the proper type.
```

Comment:
Coding standards should not be optional, there is an easy way to do this properly